Functions/xASL_stat_ComputeMean.m

```diff
@@ -12,6 +12,7 @@
 %            0 - don't do partial volume correction, just calculate a median on imMask
 %            1 - simple partial volume correction by dividing by the GM mask
 %            2 - partial volume correction using linear regression and imGM, imWM masks
```

Comment:
should be `two component PVC (otherwise known as PVC)` Perhaps also helpful to move Iris' reference to here (or refer by a * indicator)